app/common/auth/__init__.py

```diff
@@ -72,6 +73,35 @@ def claim_magic_link(magic_link_code: str) -> ResponseReturnValue:
     return render_template("common/auth/claim_magic_link.html", form=form, magic_link=magic_link)
 
 
+@auth_blueprint.route("/sso/sign-in", methods=["GET", "POST"])
+def sso_sign_in() -> ResponseReturnValue:
+    form = SSOSignInForm()
+    if form.validate_on_submit():
+        session["flow"] = build_auth_code_flow(scopes=current_app.config["MS_GRAPH_PERMISSIONS_SCOPE"])
+        return redirect(session["flow"]["auth_uri"]), 302
+    return render_template("common/auth/sign_in_sso.html", form=form)
+
+
+@auth_blueprint.route("/sso/get-token", methods=["GET"])
+@auto_commit_after_request
```

Comment:
Created https://mhclgdigital.atlassian.net/browse/FSPT-409 as a mop-up to minimise future pain from missing this